content/en/docs/reference/ide-linter-autocomplete.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "IDE linter and autocomplete for Score's JSON schema"
-linkTitle: "JSON schema autocomplete"
+title: "IDE linter for Score's JSON schema"
+linkTitle: "JSON schema IDE linter"
 weight: 10
 description: >
-  Reference documentation for the IDE linter and autocomplete for Score's JSON schema.
+  IDE linter for Score's JSON schema
```

Comment:
So behind the scene that's a JSON shema, but that's for a YAML schema from within the IDE, so where should we put `JSON schema` versus `YAML schema` in there?